integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -30,7 +28,7 @@
 
 @Tag("gravitino-docker-it")
 @TestInstance(Lifecycle.PER_CLASS)
-public class SparkIT extends SparkEnvIT {
+public class SparkCommonIT extends SparkEnvIT {
```

Comment:
could we make `SparkCommonIT` to abstract to avoid it run twice from itself and `SparkHiveCatalogIT`.

Reply:
got it. Thank you.

Reply:
updated.